core/cont/src/TClassTable.cxx

```diff
@@ -79,9 +79,6 @@ class TClassTable::NormalizeThenLock {
 
    NormalizeThenLock(const char *cname)
    {
-      if (!TClassTable::CheckClassTableInit())
```

Comment:
You modify fNormalizedName below without lock? GetNomalizedName should probably write to a temporary and only set the member once the lock is taken?

Reply:
```NormalizeThenLock``` is used as a local variable, only the currently running function in the current thread will see it.  I.e. it is already a temporary variable per se.  (On the other the mutex is a global function static).